app/controllers/miq_ae_tools_controller.rb

```diff
@@ -164,10 +164,8 @@ def import_automate_datastore
 Instances updated/added: %{instance_stats}
 Methods updated/added: %{method_stats}") % stat_options, :success)
           end
-        rescue MiqAeException::DomainNotAccessible
-          add_flash(_('Error: Selected domain is locked'), :error)
-        rescue MiqAeException::AttributeNotFound => e
-          add_flash(e.message, :error)
+        rescue => bang
+          add_flash(_("Error: %{message}" % {:message => bang.message}), :error)
```

Comment:
```_("Error: %{message}" % {:message => bang.message})```

The gettext call needs to be only around the string.

`_("Error: %{message}")` ...